updating/updating-disconnected-cluster.adoc

```diff
@@ -1,23 +1,145 @@
+to update the cluster, such as `4.4.0`.
+<2> For `<local_registry_host_name>`, specify the registry domain name for your mirror
+repository, and for `<local_registry_host_port>`, specify the port that it
+serves content on.
+<3> For `<repository_name>`, specify the name of the repository to create in your
+registry, such as `ocp4/openshift4`.
+<4> The repository to mirror. For a production release, you must specify
+`openshift-release-dev`.
+<5> For `<path_to_pull_secret>`, specify the absolute path to and file name of
+the pull secret for your mirror registry that you created.
+<6> For a production release, you must specify
+`ocp-release`.
+<7> For `server_architecture`, specify the architecture of the server e.g. x86_64.
+
+. Mirror the version images to the internal container registry.
+
+** Connect the removable media to a system connected to the internet.
+** Mirror the images and configuration manifests to a directory on the removable media.
++
+----
+$ oc adm release mirror "--to-dir=${REMOVABLE_MEDIA_PATH}/mirror" quay.io/openshift-release-dev/${RELEASE_NAME}:${OCP_RELEASE}-${ARCHITECTURE} --dry-run
+----
++
+----
+$ oc adm release mirror "--to-dir=${REMOVABLE_MEDIA_PATH}/mirror" quay.io/openshift-release-dev/${RELEASE_NAME}:${OCP_RELEASE}-${ARCHITECTURE}
+----
++
+[NOTE]
+====
+If the local container registry and the cluster connected to the mirror host, you can directly push the release images to the local registry and apply the ConfigMap to the cluster using following command.
+----
+$ oc adm release mirror --from=quay.io/openshift-release-dev/${RELEASE_NAME}:${OCP_RELEASE}-${ARCHITECTURE} \
+  --to=${LOCAL_REGISTRY}/${LOCAL_REPOSITORY} --apply-release-image-signature
+----
+====
++
+** Take the media to the restricted network environment and upload the images to the local container registry.
++
+----
+$ oc image mirror  -a ${LOCAL_SECRET_JSON} "--from-dir=${REMOVABLE_MEDIA_PATH}/mirror" file://openshift/release:${OCP_RELEASE}* ${LOCAL_REGISTRY}/${LOCAL_REPOSITORY}
+----
+
+=== Create the ConfigMap for image signature verification
+
+The Cluster Version Operator (CVO) in the cluster performs a verification to make sure the signatures of the release images are not tampered. To do that it compares expected VS actual image signatures. For the verification process it needs to know the image signatures. The image signatures are available in link:https://mirror.openshift.com/pub/openshift-v4/signatures/openshift/release[mirror.openshift.com]  and link:https://storage.googleapis.com/openshift-release/official/signatures[Google Cloud Storage (GCS)]. However in a restricted network environment we do not have access to internet, so we we need to create ConfigMap which CVO can use to get the image signatures.
+
+[NOTE]
+====
+If you are upgrading from an earlier release than 4.5.0, you must must use the manual method for creating the ConfigMap.
```

Comment:
Not sure if there is a doc's standard for this but why don't we link to the manual method adoc here and remove the 'Manual method...' section below.

Reply:
The reason I kept it separate because most likely less customers need manual method of creating the configmap as we have back ported the feature to 4.4.8. So want to keep the less text for default steps.